README 2.md

```diff
@@ -0,0 +1 @@
+# Math-Magician
```

Comment:
- [ ] Please it will be nice to update the `Getting Started` section of your readme file with explicit steps on how to clone and run your app locally as this will be of help to non technical users.